packages/tabs/src/index.tsx

```diff
@@ -5,6 +5,7 @@ export const plugins: TPlugin[] = [
   {
     pluginName: 'tabs',
     pluginType: EPluginType.UI,
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-ignore
```

I did not get any typescript errors when removing this ts ignore

Reply:
fixed